akka-stream-typed/src/test/scala/docs/akka/stream/typed/ActorSourceSinkExample.scala

```diff
@@ -5,11 +5,16 @@
 package docs.akka.stream.typed
 
 import akka.NotUsed
-import akka.stream.ActorMaterializer
+import akka.actor.typed.ActorSystem
+import akka.stream.typed.ActorMaterializer
 
 object ActorSourceSinkExample {
 
-  implicit val mat: ActorMaterializer = ???
+  val system: ActorSystem[_] = ???
+
+  implicit val mat: ActorMaterializer = ActorMaterializer()(system)
```

Comment:
yes that'll work and improves situation, I wonder though how to help Java here... Kind of important if we want traction of both java akka typed and java akka streams

Reply:
The problem is only that it has the same name, so mirroring the interface just because of that seems overkill, especially since we don't try to hide akka.stream anyway.

Perhaps we should move the typed.ActorMaterializer to scaladsl/javadsl and for javadsl rename it to `ActorMaterializerFactory` - problem solved?

Reply:
Yeah renaming it sounds better -- `akka.stream.typed.javadsl.ActorMaterializerFactory` & `akka.stream.typed.scaladsl.ActorMaterializer` not mirrored names but that's fine...

I could even see `akka.stream.typed.scaladsl.TypedActorMaterializer` work well in both javadsl and scaladsl -- it would cause less chaos during imports, and yeah both would be imported...

Reply:
I'll do that.
I think I prefer that we don't name things `TypedXxx`.

Reply:
Cool; yeah agreed on avoiding Typed in names